hypha/static_src/src/sass/apply/components/_select2.scss

```diff
@@ -68,30 +68,24 @@ $dropdown-height: 45px;
     }
 
     .select2-results__option {
-        padding: 0;
-
-        &[aria-selected='true'] {
-            .wrap {
-                &::before {
-                    background: url('./../../images/tick.svg') $color--dark-blue center no-repeat;
-                    background-size: 12px;
-                    border: 1px solid $color--dark-blue;
-                    content: '';
-                }
-            }
+        display: flex;
+        align-items: center;
+        padding: 6px;
+
+        &::before {
+            min-width: 20px;
+            height: 20px;
+            margin-right: 10px;
+            background: $color--white;
+            border: 1px solid $color--mid-grey;
+            content: '';
         }
 
-        .wrap {
-            display: flex;
-            align-items: center;
-            padding: 6px;
-
+        &[aria-selected='true'] {
             &::before {
-                min-width: 20px;
-                height: 20px;
-                margin-right: 10px;
-                background: $color--white;
-                border: 1px solid $color--mid-grey;
+                background: url('./../../images/tick.svg') $color--dark-blue center no-repeat;
+                background-size: 12px;
+                border: 1px solid $color--dark-blue;
```

Comment:
Please make the same changes in the public version of this file. Not used I think but best to keep synced.

Reply:
Done